vuepress/docs/next/tutorials/devops/entando-de-app.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebarDepth: 2
+---
+
+# Configure the Entando App Context
+This tutorial describes how to modify the [EntandoApp CR](../../docs/consume/entandoapp-cr.md) to customize the ingress path of an Entando App Engine deployment. 
+
+## Prerequisites
+- An existing deployment of an Entando App or the ability to create one.
+    - If you haven't created a deployment or don't have a YAML file for an Entando deployment, follow the [Quickstart instructions](../../docs/getting-started/README.md).
+
+## Remove or Replace entando-de-app
+
+The ingress path of the App Engine currently defaults to **/entando-de-app**, which is visible in the URL. Follow the steps below to remove or replace this value.
+
+1. Open the `entando-app.yaml`, which contains several default parameters:
+
+```yaml
+apiVersion: entando.org/v1
+kind: EntandoApp
+metadata:
+  namespace: entando
+  name: quickstart
+spec:
+  environmentVariables: null
+  dbms: embedded
+  ingressHostName: YOUR-HOST-NAME
+  standardServerImage: eap
+  replicas: 1
+```
+
+2. Under `spec`, add the parameter `ingressPath`. If this parameter is not present, the path for the `entando-de-app` defaults to **/entando-de-app**.
+
+3. Input a value for `ingressPath` to change the web context:
```

Comment:
Input a value >> enter a value or null values to... 

it doesn't explain the two different options

Reply:
hm i actually think this drives home the point that *something* needs to be entered; a null value is still a value, like 0 is still a value. if you add the parameter and leave the field blank things break

Reply:
It makes it sound as if you have to enter a value by this instruction, but the second option below says otherwise. And null value is a blank or no value. 0 is not a null value, pretty sure.

Reading the whole section I can see that there are two options. But just reading line 34, you are surprised that you can add a blank. That's all. and not a big deal

Reply:
but both 0 and null are values (you're right that 0 != null), as is a space, or empty quotes. i guess my point is that you have to add a character, even if that character is invisible. if you don't, it will error. so i didn't want to mislead and define values as only characters of a certain kind. i'll look at it again